src/features/farming/water/ProjectDignityFrogs.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext, useEffect, useState } from "react";
+import { useActor } from "@xstate/react";
+
+import * as Auth from "features/auth/lib/Provider";
+import { loadFrogs } from "features/community/merchant/actions/loadFrogs";
+import { Frog } from "features/community/project-dignity/models/frog";
+import { FrogContainer } from "features/community/project-dignity/components/FrogContainer";
+
+export const ProjectDignityFrogs: React.FC = () => {
+  const { authService } = useContext(Auth.Context);
+  const [authState] = useActor(authService);
+  const [frogData, setFrogData] = useState<Frog[]>([]);
+
+  useEffect(() => {
+    const fetchFrogs = async () => {
+      const data = await loadFrogs();
```

Comment:
Can we please get a `try { } catch {}` around this logic.

Since there are some external dependencies on APIs and other smart contracts we will need some extra fallbacks to prevent any white screens 🙏

Reply:
There is try catch block inside of loadFrogs function(covers it completely), its safe to use it.